assets/js/index.js

```diff
@@ -0,0 +1,15 @@
+// Instantiating
+var blogList = new Blogs();
+var blogs;
+var sidebar;
+
+blogList.fetch().then(function() {
```

Comment:
Instead of fetching here, move this stuff to your router's `initialize` function